dvc/command/checkout.py

```diff
@@ -1,18 +1,91 @@
 import argparse
+import logging
+
+import colorama
 
 from dvc.command.base import append_doc_link
 from dvc.command.base import CmdBase
+from dvc.exceptions import CheckoutError
+
+logger = logging.getLogger(__name__)
+
+
+def _humanize_wordseries(words=None):
```

Comment:
move it to the utils? there should be some libraries that do different "humanization" stuff ... not sure if we want to utilize them.

Reply:
I'd like to keep it near to where it's used for now. If we later need it elsewhere, we can do so at that time.

Reply:
👍 up to you, absolutely minor thing. 

Let me share my intuition behind this, though :)

- **Readability** - any boilerplate code that makes file longer and absolutely not relevant/generic makes it harder to see the "bigger picture" (tools like CodeClimate complain about file being too long, about too many methods, etc for a good reason);
- **Reusability** - the knowledge about this generic "text" util is only in your head. Low chances that someone will be able to find it when it's needed again. Means some probability of a duplicate later.
- **Testing** - you can unit test this simple things, would be strange to write a unit test for a private method though.

Reply:
It's not like humanizing, it's joining and not word series but some items in a list, so the name might be better, e.g. `human_join()` or smth.

Reply:
On keeping it here - I am with @skshetry here, move it later if that is used elsewhere. Spreading code is the big source of errors. Also impairs readability.